src/amoc_config_validation.erl

```diff
@@ -8,4 +8,9 @@ positive_integer(I) -> is_integer(I) andalso I > 0.
 
 nonnegative_integer(I) -> is_integer(I) andalso I >= 0.
 
-binary(Bitstring) -> is_binary(Bitstring).
+binary(Binary) -> is_binary(Binary).
+
+boolean(Boolean) -> is_boolean(Boolean).
```

Comment:
where this function is used? Looks like need to removed.

Reply:
It is used in `amoc_xmpp.erl`. However, for such simple functionss I would just use the built-in `is_boolean/1`.